docs/release_notes/release_notes-0.15.0.rst

```diff
@@ -13,6 +13,7 @@ Key Features and Updates
   * FIX-#4385: Get rid of `use-deprecated` option in `pip` (#4386)
   * FIX-#3527: Fix parquet partitioning issue causing negative row length partitions (#4368)
   * FIX-#4330: Override the memory limit to start ray 1.11.0 on Macs (#4335)
+  * FIX-#4273: Fix waiting of None-returning methods in benchmark mode (#4404)
```

Comment:
This is inconsistent with PR title, why?

Reply:
I agree - could we potentially change this PR's title to be more descriptive? Perhaps mention that it's a fix for benchmark mode?

Reply:
renamed the PR to match with the name in release notes